### PR TITLE
Update containers description

### PR DESCRIPTION
**Pull request by mattmgoldberg**

### CODE REVIEW

**Review by emmastephenson — APPROVED** [review]
Looks great, thanks for the cleanup Matt!

File: containers/ingestion/description.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 ```
 ❯ docker -v
 Docker version 20.10.21, build baeda1f
 ``` 
 2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/ingestion:main`.
-3. Run the service with ` docker run -p 8080:8080 ingestion:main`.
+3. Run the service with ` docker run -p 8080:8080 ghcr.io/cdcgov/phdi/ingestion:main`.
 
-Congradulations the FHIR Converter should now be running on `localhost:8080`!
+Congratulations, the FHIR Converter should now be running on `localhost:8080`!
```
Comment (DanPaseltiner) [inline containers/ingestion/description.md:21]:
```suggestion
Congratulations, the ingestion service should now be running on `localhost:8080`!
```

File: containers/message-parser/description.md [hunk 2]
```diff
@@ -11,22 +11,22 @@ The PHDI message parser is REST service for extracting desired fields from a giv
...
+1. Confirm that you have Docker installed by running `docker -v`. If you do not see a response similar to what is shown below, follow [these instructions](https://docs.docker.com/get-docker/) to install Docker.
 ```
 ❯ docker -v
 Docker version 20.10.21, build baeda1f
 ``` 
 2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/message-parser:main`.
 3. Run the service with ` docker run -p 8080:8080 message-parser:main`.
 
-Congradulations the FHIR Converter should now be running on `localhost:8080`!
+Congratulations, the FHIR Converter should now be running on `localhost:8080`!
```
Comment (DanPaseltiner) [inline containers/message-parser/description.md:29]:
```suggestion
Congratulations, the message parser should now be running on `localhost:8080`!
```

File: containers/record-linkage/description.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+1. Confirm that you have Docker installed by running `docker -v`. If you do not see a response similar to what is shown below, follow [these instructions](https://docs.docker.com/get-docker/) to install Docker.
 ```
 ❯ docker -v
 Docker version 20.10.21, build baeda1f
 ``` 
 2. Download a copy of the Docker image from the DIBBs repository by running `docker pull ghcr.io/cdcgov/phdi/record-linkage:main`.
 3. Run the service with ` docker run -p 8080:8080 record-linkage:main`.
 
-Congratulations the FHIR Converter should now be running on `localhost:8080`!
+Congratulations, the FHIR Converter should now be running on `localhost:8080`!
```
Comment (DanPaseltiner) [inline containers/record-linkage/description.md:21]:
```suggestion
Congratulations, the record linkage service should now be running on `localhost:8080`!
```

File: containers/tabulation/description.md [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
+1. Confirm that you have Docker installed by running `docker -v`. If you do not see a response similar to what is shown below, follow [these instructions](https://docs.docker.com/get-docker/) to install Docker.
 ```
 ❯ docker -v
 Docker version 20.10.21, build baeda1f
 ``` 
 2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/tabulation:main`.
 3. Run the service with ` docker run -p 8080:8080 -v $(pwd):/code tabulation:main`.
 
-Congradulations the FHIR Converter should now be running on `localhost:8080`!
+Congratulations, the FHIR Converter should now be running on `localhost:8080`!
```
Comment (DanPaseltiner) [inline containers/tabulation/description.md:21]:
```suggestion
Congratulations, the tabulation service should now be running on `localhost:8080`!
```

File: containers/validation/description.md [hunk 5]
```diff
@@ -1,45 +1,45 @@
...
+1. Confirm that you have Docker installed by running `docker -v`. If you do not see a response similar to what is shown below, follow [these instructions](https://docs.docker.com/get-docker/) to install Docker.
 ```
 ❯ docker -v
 Docker version 20.10.21, build baeda1f
 ``` 
 2. Download a copy of the Docker image from the PHDI repository by running `docker pull ghcr.io/cdcgov/phdi/validation:main`.
 3. Run the service with ` docker run -p 8080:8080 validation:main`.
 
-Congratulations the FHIR Converter should now be running on `localhost:8080`!
+Congratulations, the FHIR Converter should now be running on `localhost:8080`!
```
Comment (DanPaseltiner) [inline containers/validation/description.md:21]:
```suggestion
Congratulations, the validation service should now be running on `localhost:8080`!
```

**Review by DanPaseltiner — APPROVED** [review]
Thank you @mattmgoldberg! I just noticed a minor copy/paste error that we seemed to have tracked through all of the docs (see suggested changes).

File: containers/validation/description.md (hunk 5 above)
Comment (mattmgoldberg) [reply containers/validation/description.md:21]: Good catch!

**Pull request merged**